src/GitHub.InlineReviews/ViewModels/CommentViewModel.cs

```diff
@@ -167,6 +169,7 @@ void DoCancelEdit(object unused)
                 }
 
                 ErrorMessage = message;
+                log.Error("Error posting inline comment.", e);
```

Comment:
Just an observation. In most places we don't seem ends the message with a `.`.

It is a bit all over the place though. I've even seen one like:
`log.Error("foo {0}. ", ex);`

I guess we should do an audit at some point. 😉

Reply:
Actually i started out with no "." but then looked in the log and the messages immediately before it had a closing "." so I added it!

Reply:
lol. LGTM!